rules/do-you-confirm-there-is-no-checked-out-data/rule.md

```diff
@@ -54,75 +51,49 @@ Fill the other fields like below:
-
-<font color="#cc0000"> 
-         
-</font>To make reminding users easier, this SharePoint Add-in has a custom page to show the "Checked out files". One button will send the notification email to all the naughty people. 
+### Solution B. Custom application report (Includes some coding work)
 
+::: todo
+**TODO:** Move this tool to GitHub, find a better name than "SSW.SharePoint.CheckedOutFilesReport". Also change from a farm solution to a solution that can be used on Office365 - now in SharePoint 2016 and SharePoint online called "Sharepoint Add-ins" 
+:::
 
+To make reminding users easier, this SharePoint Add-in has a custom page to show the "Checked out files". One button will send the notification email to all the naughty people. 
 
 Even better, we have also improved the application with a scheduled task using SharePoint CSOM API to find checked out files and send these notification emails automatically every night.
 
 
-
-         <img class="ssw-rteStyle-ImageArea" alt="CheckedOutFilesApplicationReport.png" src="CheckedOutFilesApplicationReport.png">
-      <dd>Figure: One button reminds all users of their "Checked out Files"
-
-</dd>
+![Figure: One button reminds all users of their "Checked out Files"](CheckedOutFilesApplicationReport.png)
 
 ::: greybox
 
 **Hi Sophie,** 
 
- 
-
 You have some pages checked out in SharePoint.
 
 
-> 1. You should check in at least daily. Revise our SSW rule [<font color="#3a66cc">on Frequent SharePoint Check-ins</font>](/do-you-confirm-there-is-no-checked-out-data).
-> 2. If you are no longer editing these files, check them in! 
-> 
-> 3. Reply to this email with something like:
+1. You should check in at least daily. Revise our SSW rule [<font color="#3a66cc">on Frequent SharePoint Check-ins</font>](/do-you-confirm-there-is-no-checked-out-data).
+2. If you are no longer editing these files, check them in! 
+3. Reply to this email with something like:
 >     ‘Done - x files checked in’
-> 
-> 
-> 
-> 
-> You currently have the following pages checked out:
-
 
+You currently have the following pages checked out:
 
-> • 
-            <font color="#3a66cc"><a href="/rules/do-you-confirm-there-is-no-checked-out-data">http://&lt;siteurl&gt;/DesignandPresentation/RulesToBetterVideoRecording/Pages/Default.aspx</a>  (parent folder)</font>
-> • 
-            <font color="#3a66cc"><a href="/rules/do-you-confirm-there-is-no-checked-out-data">http://&lt;siteurl&gt;/DesignandPresentation/RulesToBetterVideoRecording/Pages/testing-rule.aspx</a>  (parent folder)</font>
+- http://&lt;siteurl&gt;/DesignandPresentation/RulesToBetterVideoRecording/Pages/Default.aspx  (parent folder)</font>
+- http://&lt;siteurl&gt;/DesignandPresentation/RulesToBetterVideoRecording/Pages/testing-rule.aspx  (parent folder)</font>
```

Comment:
What is the correct url for these?

Reply:
These are example URLs. I changed to use the placeholder ;)